travis/scripts/05-run.sh

```diff
@@ -47,7 +47,9 @@ launchCurlOrProtractor() {
         if [[ -f "gulpfile.js" ]]; then
             gulp itest --no-notification
         elif [[ -f "tsconfig.json" ]]; then
-            yarn e2e
+            wget -O chromedriver_linux64 https://chromedriver.storage.googleapis.com/2.37/chromedriver_linux64.zip
```

Comment:
I understand you tried to fix the strange issue with Travis, but I'm not sure about this part, because the end-user won't do this. If he need to do it, we need to document. 

I still don't understand the random failures with Travis.